playbooks/operationalizing/external-docker-registry-integration.adoc

```diff
@@ -0,0 +1,297 @@
+---
+---
+= External Docker Registry Integration with OpenShift Enterprise v3
+Andrew Block <ablock@redhat.com>
+v1.0, 2016-08-19
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+OpenShift can utilize an external docker registry as a source for deploying images and to store images produced as a result of a build. 
+
+toc::[]
+
+== Overview
+
+OpenShift is a platform for running containerized applications. Containers are instances of images and these images are stored in registries. OpenShift has the ability to leverage images stored in its own integrated registry, images stored on DockerHub or images stored in an enterprise or third party registry. This document describes the steps necessary to run containers from images stored in an external docker registry and to use an external registry to store images produced within the platform. 
+
+== Docker Configuration
+
+Regardless of the source, the underlying docker engine is used to retrieve images stored in remote repositories. There are certain considerations that must be made so that the docker engine can successfully communicate with the remote registry.
+
+=== Certificates
+
+Communication between the docker engine and the remote registry is facilitated through secure transport. A trust must be established between both parties prior to transmitting content securely. SSL certificates associated with the remote registry must be configured on on each OpenShift node that will be used to transport images. These certificates can be configured in the Red Hat Enterprise Linux (RHEL) platform truststore or for use by Docker exclusively.
+
+To configure docker to trust the certificate of a remote repository that is not included in the platform trust store, create a folder within the `/etc/docker/certs.d` folder with the name and optional port value. If the remote registry had the address of `external-registry.example.com:5000` the folder path would be `/etc/docker/certs.d/external-registry.example.com`.
+
+Copy the certificate into this folder and rename the file *ca.crt*
+
+=== Insecure Registry
+
+Even though Docker and OpenShift emphasize secure communication, the Docker daemon can be configured to retrieve images from insecure registries. Insecure registries can refer to ones that are protected with self signed or untrusted SSL certificates or ones that do not communicate over secure channels (HTTP). 
```

Comment:
The last part of the final sentence describes the use of an insecure communication channel (ie HTTP). Existing content should be valid